### DISCUSSION TITLE
Chronos GPT-2 base model checkpoint missing

### DISCUSSION THREAD

**Discussion by oops343** [amazon-science/chronos-forecasting#196] [Q&A] [answered]
thanks for the excellent work
the gpt2 model checkpoint is not found in [huggingface link](https://huggingface.co/collections/amazon/chronos-models-and-datasets-65f1791d630a8d57cb718444) provided in README, so is there any plan on releasing this ckpt?

**Comment by abdulfatir** [comment, marked as answer]
Hi @oops343! We don't plan to release the GPT-2 checkpoint but all the code and data to pretrain your own model is available in this repo. Let me know if you have questions.

**Reply by oops343** [reply]
I noticed that the dataset in hf is not compatable with the current training script... not sure if this is a problem, but do I need to convert the ~100G files into gluonts formats or is there any workaround, which would be appreciated ofc ;）

**Reply by abdulfatir** [reply]
@oops343 Yes, I understand that this may be a little bit annoying. We are tracking this issue in #194. Of course, we welcome your contributions to enable this, if you have the bandwidth.
